### PR TITLE
Feature/dashboard dashboard

### PR DESCRIPTION
**Pull request by tobyhede**

### CODE REVIEW

**Comment by elisechant** [comment]
beautiful feature idea

**Comment by catherine-jones** [comment]
@tobyhede what's going on with this branch is it in progress should we put a label on it

**Comment by micapam** [comment]
Not enough dashboards in the PR name. LBTM

File: spec/decorators/dashboard_decorator_spec.rb [hunk 1]
```diff
@@ -59,6 +59,12 @@
 
       it { is_expected.to eq 'Blah Dashboard' }
     end
+    context 'dashboard dashboard' do
+      let(:opts) { { :name => 'Blah Dashboard Dashboard' } }
+      subject { decorator.dashboardized_name }
+
+      it { is_expected.to eq 'Blah Dashboard' }
```
Comment (elisechant) [inline spec/decorators/dashboard_decorator_spec.rb]: truly amazing test here

**Comment by elisechant** [comment]
closing because new squashed commit opened here https://github.com/AusDTO/dto-dashboard/pull/318

**Pull request closed without merging**